scripts/find_max_intensity_diffusion_path.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+
+from quactography.graph.graph_with_connexions import Graph
+from quactography.adj_matrix.io import load_graph
+from quactography.hamiltonian.hamiltonian_total import Hamiltonian
+from quactography.solver.multiprocess_solver import multiprocess_qaoa_solver
+from quactography.solver.qaoa_solver import _find_longest_path
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter
+    )
+    p.add_argument(
+        "in_graph",
+        help="Adjacency matrix which graph we want path that maximizes weights in graph, npz file",
+        type=str,
+    )
+    p.add_argument("starting_node", help="Starting node of the graph", type=int)
+    p.add_argument("ending_node", help="Ending node of the graph", type=int)
+    p.add_argument("output_file", help="Output file name", type=str)
+
+    p.add_argument(
+        "--alphas", nargs="+", type=int, help="List of alphas", default=[1.1]
+    )
+    p.add_argument("--multiprocess", help="Use multiprocess", action="store_true")
+
+    p.add_argument(
+        "--reps",
+        help="Number of repetitions for the QAOA algorithm",
+        type=int,
+        default=1,
+    )
+    # Voir avec scilpy :
+    p.add_argument(
```

Comment:
I would suggest using the `add_processes_arg` from `scilpy.io.utils` instead. Also, you don't need the `--multiprocess` argument. See my comment for `solver.qaoa_solver.py`.